experiments/AtmosLES/bomex.jl

```diff
@@ -0,0 +1,513 @@
+                                   bc::BOMEX_BC, atmos::AtmosModel,
+                                   F,
+                                   state⁺, diff⁺, aux⁺, n⁻,
+                                   state⁻, diff⁻, aux⁻,
+                                   bctype, t,
+                                   state₁⁻, diff₁⁻, aux₁⁻)
+When `bctype == 1` the `NoFluxBC` otherwise the specialized BOMEX BC is used
+"""
+#TODO This needs to be in sync with the new boundary condition interfaces
+function atmos_boundary_flux_diffusive!(nf::CentralNumericalFluxDiffusive,
+                                        bc::BOMEX_BC, 
+                                        atmos::AtmosModel, F,
+                                        state⁺, diff⁺, hyperdiff⁺, aux⁺,
+                                        n⁻,
+                                        state⁻, diff⁻, hyperdiff⁻, aux⁻,
+                                        bctype, t,
+                                        state₁⁻, diff₁⁻, aux₁⁻)
+  
+  # Floatint point precision
+  FT = eltype(state⁺)
+
+  # Establish the thermodynamic state based on the prognostic variable state
+  TS = thermo_state(atmos.moisture, atmos.orientation, state⁺, aux⁺)
+
+  # Air temperature given current thermodynamic state
+  temperature = air_temperature(TS)
+
+  # Boundary condition for bottom wall
+  if bctype != 1
+    atmos_boundary_flux_diffusive!(nf, NoFluxBC(), atmos, F,
+                                   state⁺, diff⁺, hyperdiff⁺, aux⁺, n⁻,
+                                   state⁻, diff⁻, hyperdiff⁻, aux⁻,
+                                   bctype, t,
+                                   state₁⁻, diff₁⁻, aux₁⁻)
+  else
+    # Start with the noflux BC and then build custom flux from there
+    atmos_boundary_state!(nf, NoFluxBC(), atmos,
+                          state⁺, diff⁺, aux⁺, n⁻,
+                          state⁻, diff⁻, aux⁻,
+                          bctype, t)
+
+    # Interior state velocities [u₀]
+    u₁ = state₁⁻.ρu / state₁⁻.ρ
+    # Windspeed at first interior node
+    # [Impenetrable flow, normal component of velocity is zero, horizontal components may be non-zero]
+    windspeed₁ = norm(u₁)
+    # Turbulence tensors in the interior state
+    _, τ⁻ = turbulence_tensors(atmos.turbulence, state⁻, diff⁻, aux⁻, t)
+    u_star = bc.u_star # Constant value for friction-velocity u_star == u_star
+    @inbounds begin
+      τ13⁺ = -u_star^2 * windspeed₁[1] / norm(windspeed₁) # ⟨u′w′⟩ 
+      τ23⁺ = -u_star^2 * windspeed₁[2] / norm(windspeed₁) # ⟨v′w′⟩
+      τ21⁺ = τ⁻[2,1]
+    end
+    # Momentum boundary condition [This is topography specific #FIXME: Simon's interface will improve this]
+    τ⁺ = SHermitianCompact{3, FT, 6}(SVector(0   ,
+                                             τ21⁺, τ13⁺,
+                                             0   , τ23⁺, 0))
+    # Moisture boundary condition ⟨w′qt′⟩
+    d_q_tot⁺  = SVector(0, 
+                        0, 
+                        state⁺.ρ * bc.w′qt′)
```

Comment:
```suggestion
                        LHF / latent_heat_vapor(T_sfc))
```
(Something like this. I'm not sure whether we can access T_sfc here.)

Reply:
`TS` a few lines above establishes the boundary thermodynamic state, from there we can get `air_temperature(TS)` which makes this a reasonable expression.

Reply:
I would like to have the prescribed surface temperature (300.4K) here if possible, because it would make the resulting w′qt′ consistent with the paper. But if it is easier to use the surface air temperature (or first model level temperature?) here, that sounds good to me too.